### PR TITLE
fix: add experimental feature note to TTFD docs

### PR DESCRIPTION
**Pull request by armcknight**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs
- [x] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

*Describe your changes here. If your PR relates to or resolves an issue, add a link to that too.*

## Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Review by philipphofmann — APPROVED** [review]
LGTM

File: src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 1]
```diff
@@ -339,6 +339,8 @@ The span finishes after the view appeared on the screen.
 
 ## Time to Full Display
 
+<Note>This feature is experimental and may have bugs.</Note>
```
Comment (vivianyentran) [inline src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx:342]:
```suggestion
<Note>

This feature is experimental and may have bugs.

</Note>
```
There's been some rendering issues when the extra lines between the note tags aren't added, so let's just be safe here! This won't be an issue in the future when the docs move to next.js though!

**Review by vivianyentran — APPROVED** [review]
adjusted formatting, but otherwise LGTM

File: src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx (hunk 1 above)
Comment (armcknight) [reply src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx:342]: Thanks for the heads up!

**Pull request merged**